core/tests/protractor_utils/CommunityDashboardPage.js

```diff
@@ -24,10 +24,59 @@ var CommunityDashboardPage = function() {
   var navigateToTranslateTextTabButton = element(
     by.css('.protractor-test-translateTextTab'));
 
+  var reviewRightsDiv = element(by.css('.protractor-test-review-rights'));
+
+  this.get = function() {
+    browser.get('/community_dashboard');
+    waitFor.pageToFullyLoad();
+  };
+
   this.getTranslateTextTab = function() {
     return new CommunityDashboardTranslateTextTab
       .CommunityDashboardTranslateTextTab();
   };
+
+  this.expectUserToBeTranslationReviewer = function(language) {
+    waitFor.visibilityOf(
+      reviewRightsDiv, 'User does not have rights to review translation');
```

Comment:
We often see test failures/flakes because these `waitFor` calls time out, and sometimes it's because of a slow machine or the test being on the wrong page. In these cases, this error message could be confusing because it suggests that the user actually does not have rights even though the issue might instead be with the page loading. How about phrasing the error message as an element failing to load instead?

Reply:
Also see my last comment for more thoughts on how to phrase this message

Reply:
If the element `reviewRightsDiv` is not visible then the user doesn't have the rights to review stuff. I'm not sure whether we should say `failing to load`, let me know what you think?

Reply:
My concern is that there are two potential causes for `reviewRightsDiv` not being visible:
1. The user doesn't have rights to review stuff
2. The page didn't load properly

Is there a way to phrase the error message to communicate that both of these are possible?